gradle/libs.versions.toml

```diff
@@ -128,6 +132,12 @@ treemapfx = { module = "software.coley:treemap-fx", version.ref = "treemapfx" }
 
 vineflower = { module = "org.vineflower:vineflower", version.ref = "vineflower" }
 
+fernflower = { module = "com.jetbrains.intellij.java:java-decompiler-engine", version.ref = "fernflower" }
+
+forgeflower = { module = "net.minecraftforge:forgeflower", version.ref = "forgeflower" }
+
+annotations = { module = "org.jetbrains:annotations", version.ref = "annotations" }
```

Comment:
Why were JetBrains annotations added?

Reply:
This is a dependency of fernflower and forgeflower, and may not be a necessary dependency. I just tested it and it has no effect if I remove it.

Reply:
I will do more testing and decide whether to remove it.

Reply:
I have tested many times, and the dependence has no effect. Now he has been removed.